CHANGELOG.rst

```diff
@@ -2,6 +2,201 @@
 Release Notes
 =============
 
+.. _Release Notes_7.37.0:
+
+7.37.0 / 6.37.0
+======
+
+.. _Release Notes_7.37.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2022-06-27
+
+- Please refer to the `7.37.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7370>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.37.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- OTLP ingest: Support for the deprecated ``experimental.otlp`` section and the ``DD_OTLP_GRPC_PORT`` and ``DD_OTLP_HTTP_PORT`` environment variables has been removed. Use the ``otlp_config`` section or the ``DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT`` and ``DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT`` environment variables instead.
+
+- OTLP: Deprecated settings ``otlp_config.metrics.report_quantiles`` and ``otlp_config.metrics.send_monotonic_counter`` have been removed in favor of ``otlp_config.metrics.summaries.mode`` and ``otlp_config.metrics.sums.cumulative_monotonic_mode`` respectively.
+
+
+.. _Release Notes_7.37.0_New Features:
+
+New Features
+------------
+
+- Adds User-level service unit filtering support for Journald log collection via `include_user_units` and `exclude_user_units`.
```

Comment:
```suggestion
- Adds User-level service unit filtering support for Journald log collection via ``include_user_units`` and ``exclude_user_units``.
```

I won't add a comment for others but there's a bunch of single backticks where there should be a single one. I think we hit this issue enough with reno release notes that maybe some automation around it would be good.